tasks-by-users/AbbasShaikh0304/p1/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+  <html>
+      <head>
+          <meta charset="utf-8">
+          <meta name="keywords" content="Automobile, parts, garage">
+          <meta name="Description" content="Welcome to our garage to get parts at most affordable prizes">
+          <title>Welcome to Online Book Store</title>
+      </head>
+      <body>
+          <header><img src="/img/Logo.png" width="100%" height="150px" alt="oops!"></header>
+          <h1>
```

Comment:
why is this resolved? You should not resolve until you fixed it @AbbasShaikh0304